### ISSUE TITLE
Design Intent - Accessibility Feedback - Authenticated Experience, ILER IES PDF Download

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#109284]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

- Overall I think this all makes sense!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Sooner rather than later, reach out to the ADE team (cc @artsymartha68) to get help with PDF accessibility. If what you're getting from DoD is good then great! If it's got issues then the ADE folks should be able to identify them and you can coordinate with DoD.
- [ ] **Coding note:** PDF downloads are the biggest friction point of "buttons vs links," but the decision largely comes down to this: Will clicking the download element almost-immediately start the download (ie. roughly what you experience with a link)? Then use the download variant of `va-link`. Will clicking the download element lead to some waiting period before the start of the download (ie. long enough that you might need a loading indicator)? Then use the `va-button`.
   
   You may not have the answer to this at this point in your product lifecycle, but I'm mentioning it now to get it on your radar so you're aware that you may need to swap the link for a button depending on how the technical side of it plays out.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by lakper** [comment]
@mtcA6 @jfinnerty13 I am the Delivery Lead on the Accessibility Digital Experience team working with @artsymartha68  . When you are ready with either a sample PDF or the final deliverable let us know, we can review it to ensure it is accessible and meets 508 requirements as well as help you prepare for your next Collaboration Cycle checkpoints.
CC: @EvanAtCoforma

**Comment by mtcA6** [comment]
@lakper sounds great! We're quite a ways off from connecting to ILER once we've done that then we can fix our attention on the pdf.

**Comment by mtcA6** [comment]
@jfinnerty13 i think we should update designs to use the button as recommended and then could close this out until we've connected to ILER. The must feedback isn't actionable/pertinent until we get to midpoint or staging when we're actually connected and able to generate some test data.

**Comment by jfinnerty13** [comment]
Noting I see the feedback and will make the updates accordingly once we have more information. Thanks @briandeconinck !
